crates/typst/src/eval/methods.rs

```diff
@@ -147,6 +147,7 @@ pub fn call(
             "sorted" => array.sorted(vm, span, args.named("key")?)?.into_value(),
             "zip" => array.zip(args.expect("other")?).into_value(),
             "enumerate" => array.enumerate().into_value(),
+            "dedup" => array.dedup(vm, args.named_or_find("key")?)?.into_value(),
```

Comment:
```suggestion
            "dedup" => array.dedup(vm, args.named("key")?)?.into_value(),
```
Just noticed this. I'd prefer not allowing the shorthand here for consistency with `sorted`. Also, if we decide to add `by` to both in the future, it would be ambigious.